okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -49,8 +48,8 @@
     NONE,
     /**
      * Logs request and response lines.
-     * <p>
```

Comment:
When `<p>` is on its own line, IntelliJ thinks the line break is accidental and it collapses everything into a single (wrapped) line. With a blank line between the paragraphs, IntelliJ autoformatting preserves that blank line.